hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -302,7 +302,16 @@ public AbfsRestOperation createPath(final String path, final boolean isFile, fin
             HTTP_METHOD_PUT,
             url,
             requestHeaders);
-    op.execute();
+    try {
+      op.execute();
+    } catch (AzureBlobFileSystemException ex) {
+      String existingResource =
```

Comment:
first check if the httpstatus code is 409 and (!isFile), then retrieve the existingResource header.

Reply:
done